src/libraries/System.Net.Quic/src/System/Net/Quic/QuicError.cs

```diff
@@ -72,5 +72,10 @@ public enum QuicError
         /// The operation has been aborted.
         /// </summary>
         OperationAborted,
+
+        /// <summary>
+        /// OS does not support IPv6
+        /// </summary>
+        OSDoesNotSupportIPv6
```

Comment:
does EADDRNOTAVAIL always mean IPv6 not supported? can the same error occur for Ipv4? I think we should call it "AddressNotAvailable" so that we don't assume one specific reason for the error code.

Reply:
It's for cases where local addresss is not available for some reason (like when urls can not be resolved ), it's not just for when using IPv6/IPv4 with a machine that does not support it